grid/chat/members/maps/discord.suma

```diff
@@ -0,0 +1,187 @@
+profile = "chat/members@1.0"
+provider = "discord"
+
+"""
+GetMembers map
+"""
+map GetMembers {
+  return map error if (input.workspace === undefined) {
+    title = "Input parameter `workspace` is required for provider discord"
+  }
+
+  http GET "/guilds/{input.workspace}/members" {
+    request "application/json" {
+      query {
+        after = input.page
+        limit = input.limit
+      }
+
+      headers {
+        "Authorization" = `Bot ${parameters.accessToken}`
+      }
+    }
+
+    response 200 "application/json" {
+      members = call MapMembers(members = body)
+      rateLimit = call MapRateLimitDetails(headers = headers)
+
+      result.members = members,
+      result.nextPage = members[members.length - 1].id
```

Comment:
This way I will either hit an error (if the array is empty), or the pagination will never end.